polymesh_schema.json

```diff
@@ -1,7 +1,7 @@
 {
     "types": {
         "IdentityId": "[u8; 32]",
-        "InvestorUid": "[u8; 32]",
+        "InvestorUid": "[u8; 16]",
```

Comment:
Hmm, Is 128 bits of entropy enough for our use-case?

Reply:
This is the output of PUIS. I'm tagging @JesseALancaster to comment.